regulation-worker/internal/client/client.go

```diff
@@ -50,8 +50,9 @@ func (j *JobAPI) Get(ctx context.Context) (model.Job, error) {
 	}
 	req.SetBasicAuth(j.Identity.BasicAuth())
 	req.Header.Set("Content-Type", "application/json")
-
+	reqTime := time.Now()
 	resp, err := j.Client.Do(req)
+	stats.Default.NewStat("regulation_manager.request_time", stats.TimerType).Since(reqTime)
```

Comment:
either use a separate measurement name, or use a tag to denote the request type

Reply:
added it as tag.